src/main/java/core/basesyntax/model/TypeActivity.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum TypeActivity {
+    BALANCE("b"), SUPPLY("s"), PURCHASE("p"), RETURN("r");
+
+    private static final Map<String, TypeActivity> BY_LABEL = new HashMap<>();
```

Comment:
use `values()` instead of this hash map, it will simplify your code

Reply:
I disagree, this way can not help me convert label to typeActivity